lesson_1/phone.py

```diff
@@ -0,0 +1,58 @@
+class Phone:
+    _counter: int = 0
+    description: str = "Nobody"
+    ph_number: str = "+380000000000"
+
+    def change_number(self, ph_number) -> str:
```

Comment:
Що користуєшся тайп хінтінгами - добре.
Погано - що робиш те, як попало. Мало написати `-> str`, треба щоб воно відповідало дійсності.
```suggestion
    def change_number(self, ph_number: str) -> None:
```